tidb-cloud/size-your-cluster.md

```diff
@@ -128,7 +131,9 @@ TiFlash supports up to 2 TiB storage capacity.
 
 ## Performance reference
 
-This section provides [TPC-C](https://www.tpc.org/tpcc/) and [Sysbench](https://github.com/akopytov/sysbench) performance test results of five popular TiDB cluster scales, which can be taken as a reference when you determine the cluster size.
+This section provides [TPC-C](https://www.tpc.org/tpcc/) and [Sysbench](https://github.com/akopytov/sysbench) performance test results of several popular TiDB cluster scales, which can be taken as a reference when you determine the cluster size.
+
+### 4 vCPU, 8 vCPU, and 16 vCPU performance references
```

Comment:
suggest:

### 2 vCPU performance
### 4 vCPU performance
### 8 vCPU performance
### 16 vCPU performance

Reply:
The test environment of 2 vCPU is different from others, so we might need to consider how we arrange this information. Do you think that we could improve this in another PR?